### PR TITLE
Try ruby version

### PR DESCRIPTION
**Pull request by sachinsaxena1996**
Remove ruby version dependency from gemspec and mute rubocop which checks it

### CODE REVIEW

File: digest_generator.gemspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
                           to be compatible with postgres bigints, supports xxHash32 and xxHash64 also'
   spec.homepage      = 'https://github.com/YourMechanic/digest_generator'
   spec.license       = 'MIT'
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.6.2')
+  # spec.required_ruby_version = Gem::Requirement.new('>= 2.3.0')
```
Comment (siddyg) [inline digest_generator.gemspec:16]: any reason this is commented out?

File: .ruby-version [hunk 2]
```diff
@@ -1 +0,0 @@
-2.6.2
```
Comment (siddyg) [inline .ruby-version:1]: don't we need this file with `2.3.0?`

File: digest_generator.gemspec (hunk 1 above)
Comment (sachinsaxena1996) [reply digest_generator.gemspec:16]: @siddyg required ruby version 2.3.0 has rubocop warning we saw in slack. So I removed required ruby version and documented the rubocop in todo file.

File: .ruby-version (hunk 2 above)
Comment (sachinsaxena1996) [reply .ruby-version:1]: not required

**Review by siddyg — APPROVED** [review]

**Comment by siddyg** [comment]
@sachinsaxena1996 we need to do a gem version update too as part of this PR and push to rubygem right?

**Pull request merged**